csrc/src/lib.rs

```diff
@@ -280,6 +289,7 @@ impl FlashAttention {
                 /* window_size_right */ window_size_right,
                 /* softcap */ softcap,
                 false,
```

Comment:
all of the parameters above have the argument name in the comment, maybe add these two as well?

Reply:
addressed